extension/background.js

```diff
@@ -1,79 +1,107 @@
-username = "";
+var username = "";
```

Comment:
Does this actually change the scope in this case? I think your addition is more correct but I can't remember if JS attaches the `username` on to `window` or if `var username` attaches the variable to a more local scope?